core/Convert.php

```diff
@@ -353,4 +355,19 @@ public static function raw2url($title) {
 	public static function nl2os($data, $nl = PHP_EOL) {
 		return preg_replace('~\R~u', $nl, $data);
 	}
+
+	/*
+	 * Convert a FQCN to be suitable for an HTML attribute.
+	 *
+	 * Replaces backslashes (\) with dashes (-) then passes
+	 * the resulting string through Convert::raw2htmlname().
+	 *
+	 * @uses Convert::raw2htmlname()
+	 * @param string $val String to escape
+	 * @return string
+	 */
+	public function fqcn2htmlname($val)
+	{
```

Comment:
This is pedantic, but the coding conventions require the `{` to be on the same line as the `)`